### DISCUSSION TITLE
Feature request - autocomplete for literals and `in` operator

### DISCUSSION THREAD

**Discussion by Andrej730** [microsoft/pylance-release#6064] [Enhancement]
My suggestion is to add an autocomplete for literals and `in` operator similar to how it works with `==` operator already.

Now when you type `if v == "`, Pylance offers autocomplete for literal values available for `v` (a / b / c / d)
The same way it would work when you type `v in ("` or `v in ["` and it would show a dropdown with b / c / d values.

The main benefit for this that it will be easier to type a check to narrow down a literal type and it will also will be consistent with autocomplete for `== "`.

https://github.com/microsoft/pylance-release/assets/9417531/9f7da38c-ebd8-4dae-b84e-cb403dd574d4

Test snippet:
```python
from typing import Literal
Test = Literal["a", "b", "c", "d"]

def test(v: Test):
    if v == "a":
        pass
    elif v in ("b", "c"):
        pass
```

**Comment by heejaechang** [comment]
moved to discussion from issue to gather upvotes.
